code/modules/projectiles/ammunition/energy/stun.dm

```diff
@@ -1,19 +1,22 @@
 /obj/item/ammo_casing/energy/electrode
 	projectile_type = /obj/projectile/energy/electrode
-	select_name = "stun"
+	select_name = "tasing"
 	fire_sound = 'sound/weapons/taser.ogg'
-	e_cost = 200
+	e_cost = 100

```

Comment:
Did you make sure all the guns that fire this ammo needs twice as many shots?

Reply:
what do you mean? halving the cost for it should mean all guns can fire double the amount of tases?

Reply:
Right, and if a gun can already fire 10 tasers it probably shouldnt be able to now fire 20. You are trying to weaken the taser, giving it the same total stopping power doesn't make sense in all cases. So please assess, and report your findings.